app/modules/sightings/models.py

```diff
@@ -30,7 +30,7 @@ def __repr__(self):
 
     def get_owners(self):
         owners = []
-        for encounter in self.get_encounters():
+        for encounter in self.owned_encounters:
```

Comment:
where is `owned_encounters` coming from here?  maybe i just got lost, but i thought we only had **user**.owned_encounters as a backref

Reply:
Wot @naknomum  said. I just searched for this and I can't find it on main which does beg the question of how on earth doesn't this fall over?

Reply:
I think I got my wires crossed here editing User.get_sightings